js/tipser.js

```diff
@@ -11,9 +11,11 @@ const TipserProductView = Backbone.View.extend({
     click: 'handleClick'
   },
   handleClick: function(e) {
+    e.stopPropagation();
```

Comment:
Replace with `e.preventDefault` and move `const` delcarations to top of function.